deploy/sample-backup-configs/sample_config_runtime.yaml

```diff
@@ -1,17 +1,13 @@
 # RUNTIME configuration for the FireSim Simulation Manager
 # See https://docs.fires.im/en/stable/Advanced-Usage/Manager/Manager-Configuration-Files.html for documentation of all of these params.
 
-run_farm:
-    run_farm_tag: mainrunfarm
-    always_expand_runfarm: yes
-    f1_16xlarges: 1
-    m4_16xlarges: 0
-    f1_4xlarges: 0
-    f1_2xlarges: 0
-    launch_instances_timeout_minutes: 60
-    run_instance_market: ondemand
-    spot_interruption_behavior: terminate
-    spot_max_price: ondemand
+run_farm_config:
+  # managerinit replace start
+  defaults: sample-run-farm-recipes/aws_ec2.yaml
+  # Uncomment and add args to override defaults
+  #override_args:
+  #  <ARG>: <OVERRIDE>
```

Comment:
See https://github.com/firesim/firesim/pull/1028/files#r885857553 for how overrides work.